tests/test_preprocess.py

```diff
@@ -1,24 +1,24 @@
 from tempfile import mkdtemp
 from shutil import rmtree
 import os
+from xml.dom.minicompat import defproperty
```

Comment:
I have no recollection at all of adding this.
Just remove it, it is not used anywhere